Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -45,9 +45,10 @@ public extension VisualInstruction {
          
          - parameter image: The component’s preferred image representation.
          - parameter alternativeText: The component’s alternative text representation. Use this representation if the image representation is unavailable or unusable, but consider formatting the text in a special way to distinguish it from an ordinary `.text` component.
+         - parameter shield: Optionally, a structured image representation for displaying a [highway shield](https://en.wikipedia.org/wiki/Highway_shield).
          */
-        case image(image: ImageRepresentation, alternativeText: TextRepresentation)
-        
+        case image(image: ImageRepresentation, alternativeText: TextRepresentation, shield: ShieldRepresentation? = nil)
```

Comment:
@bamx23 pointed out that, even with the default case, a switch statement will break because of this extra argument. We’ll need to move this into a separate case for when there is a shield representation.

Reply:
#646